dewy/migrations/0001_schema.sql

```diff
@@ -31,7 +31,7 @@ CREATE TYPE ingest_state AS ENUM ('pending', 'ingested', 'failed');
 CREATE TABLE document(
     id SERIAL NOT NULL,
     collection_id INTEGER,
-    url VARCHAR NOT NULL,
+    url VARCHAR,
```

Comment:
Could we call this "name" and use either URL or file path?

Reply:
Maybe? I'd be more inclined to add a separate "title" path. It feels weird that "name" is overloaded. And what about cases where a document is created with a given URL, and then updated with content, etc.

I actually think we may want to have a separate `document_version` table at some point, but not sure (seems more useful to just retain the latest here). Also not sure we want to rely on the path for anything, since it can be set by the client arbitrarily.

Reply:
Moved to #83.

Reply:
Yeah I was thinking about versions too. Probably not something we need to solve right now though (at minimum we need to implement update)